subsys/bluetooth/controller/ll_sw/lll_conn_iso.h

```diff
@@ -58,6 +58,11 @@ struct lll_conn_iso_stream {
 	uint8_t datapath_ready_rx:1;/* 1 if datapath for RX is ready */
 
 #if !defined(CONFIG_BT_CTLR_JIT_SCHEDULING)
+	/* A CIS LLL is active and established in a CIG radio event when ACL instant has passed
+	 * which set the `active` flag and then when CIG event prepare has picked up the set
+	 * `active` flag.
+	 */
+	uint8_t established:1;
```

Comment:
If I understand the usage of this flag correctly it is basically used as an "was active when prepare was called". Naming it `established` is misleading since that has a very specific meaning in the BT spec; Wouldn't `active_prepare` work better (that seems like the usual naming convention)?

Reply:
agree.. how about just `cis->prepared` ?

Reply:
Makes sense :+1: